frontend/src/metabase/core/components/FormField/FormField.styled.tsx

```diff
@@ -1,15 +1,17 @@
 import styled from "@emotion/styled";
-import React from "react";
 import { color } from "metabase/lib/colors";
 import Icon from "metabase/components/Icon";
 import { FieldAlignment, FieldOrientation } from "./types";
 
 export interface FormCaptionProps {
   alignment: FieldAlignment;
   orientation: FieldOrientation;
+  hasDescription: boolean;
 }
 
 export const FieldCaption = styled.div<FormCaptionProps>`
+  align-self: ${props =>
+    props.orientation !== "vertical" && !props.hasDescription ? "center" : ""};
```

Comment:
I think this could be clearer since `orientation` can only either be `vertical` or `horizontal`.

What about checking if this is `horizontal` instead? Not equal kind of implies there could be more than 2 values. What do you think?

Reply:
Also nit: other CSS properties inside `<FieldCaption />` seem to use another pattern `predicates && values` versus `predicates ? values: ""`. I think we should make it consistent.